File: backend/app/schemas/mensaje.py
from pydantic import BaseModel, Field
from typing import Optional
from datetime import datetime

class MensajeCreate(BaseModel):
    contenido: str = Field(..., min_length=1, max_length=2000)

class MensajeResponse(BaseModel):
    id: int
    incidencia_id: int
    usuario_id: int
    usuario_nombre: str
    usuario_rol: str
    contenido: str
    creado_en: datetime
    
    class Config:
        from_attributes = True

File: mobile/app/src/main/java/com/tomoko/fyntra/data/models/Mensaje.kt
package com.tomoko.fyntra.data.models

data class Mensaje(
    val id: Int,
    val incidencia_id: Int,
    val usuario_id: Int,
    val usuario_nombre: String,
    val usuario_rol: String,
    val contenido: String,
    val creado_en: String
)

data class MensajeCreate(
    val contenido: String
)

